graphql2/schema.graphql

```diff
@@ -601,6 +614,31 @@ input CreateGQLAPIKeyInput {
   role: UserRole!
 }
 
+input UpdateGQLAPIKeyInput {
+  id: ID!
+  name: String
+  description: String
+}
+
+type GQLAPIKey {
+  id: ID!
+  name: String!
+  description: String!
+  createdAt: ISOTimestamp!
+  createdBy: User @goField(forceResolver: true)
+  updatedAt: ISOTimestamp!
+  updatedBy: User @goField(forceResolver: true)
```

Comment:
The resolver stuff is backend/behind the scenes/implementation details, if we add a comment here it will show up in the end-user API documentation which would be confusing since from the API side a User is returned.

`goField` directives themselves are not exposed since they configure gqlgen, but comments are used for the API.